README.md

```
@@ -126,12 +126,12 @@ VM -->|"Uses"| UC
 ## 🚦 Getting Started
 
 1. **Prerequisites:**
-    - Android Studio Ladybug (or newer)
+    - Android Studio
     - JDK 17+
 
 2. **Clone & Run:**
     ```bash
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)